### PR TITLE
add household records

### PR DESCRIPTION
**Pull request by XueliangWang**

### CODE REVIEW

File: Matching/cpsmar.py [hunk 1]
```diff
@@ -20,19 +20,135 @@ def h_recs(rec):
     """
     record = pd.DataFrame()
     record['hrecord'] = [int(rec[0])]
-    record['h_seq'] = [int(rec[1:6])]
-    record['hhpos'] = [int(rec[6:8])]
+    record['h_seq'] = [int(rec[1:5])]
+    record['hhpos'] = [int(rec[6:7])]
```
Comment (andersonfrailey) [inline Matching/cpsmar.py]: @XueliangWang, before I merge can you change all of these back to the numbers they were previously. The way python indexing works the character at the last number is not included.

For example, say you have a string `x = '1234'` and want to get the first three characters (`x[0] = 1, x[1] = 2, x[2] = 3`).
Only using `x[0:2]` will get you `'12'`. To get all three you need `x[0:3]`:
`x[0:3] = '123'`

So with your changes the final character will be inadvertently cut off. You many also need to add one to the rest of your numbers to ensure they are correct.

**Comment by andersonfrailey** [comment]
@XueliangWang, thanks for the quick fixes! Merging. Can you add two new functions for family and person records now and when those are complete open up a new pull request.

**Comment by XueliangWang** [comment]
Sure

**Comment by XueliangWang** [comment]
@andersonfrailey  Sure

**Pull request merged**